lxd/instance/drivers/driver_qemu.go

```diff
@@ -1402,8 +1438,9 @@ WantedBy=multi-user.target
 Description=LXD - agent - 9p mount
 Documentation=https://linuxcontainers.org/lxd
 ConditionPathExists=/dev/virtio-ports/org.linuxcontainers.lxd
-After=local-fs.target
+After=local-fs.target lxd-agent-virtiofs.service
 DefaultDependencies=no
+ConditionPathExists=!/run/lxd_config/9p
```

Comment:
Probably should move away from calling the directory `9p`. Instead maybe just call it `drive`?